lib/fencing/st_client.c

```diff
@@ -905,15 +905,20 @@ invoke_registered_callbacks(stonith_t *stonith, xmlNode *msg, int call_id)
     if (msg == NULL) {
         // Fencer didn't reply in time
         rc = -ETIME;
+        CRM_LOG_ASSERT(call_id > 0);
 
     } else {
         // We have the fencer reply
 
-        crm_element_value_int(msg, F_STONITH_RC, &rc);
-        crm_element_value_int(msg, F_STONITH_CALLID, &call_id);
-    }
+        if (crm_element_value_int(msg, F_STONITH_RC, &rc) != 0) {
```

Comment:
whitespace?

Reply:
Unless I'm missing something, it's correct -- the closing brace got moved down a bit (it's now checking the return value of crm_element_value_int() where it wasn't before)